docs/manual-testing-playbook.md

```diff
@@ -0,0 +1,631 @@
+```
+
+Verify you see an HTML response of a directory listing including the `test-file-1.txt` created above.
+
+Try to navigate to the https tailnet url from your local web browser (on a machine connected to the internet, but not to the tailnet),
+and verify that it is not accessible.
+
+### Tailscale funnel
+
+`tailscale funnel` acts as a reverse proxy for a local HTTP/S server,
+to expose it securely to the public internet.
+
+To test this, run this example webserver in one terminal:
+
+```bash
+lxc exec $TAILSCALE_VM_1 -- mkdir tmp-server2
+lxc exec $TAILSCALE_VM_1 -- touch tmp-server2/test-file-2.txt
+lxc exec $TAILSCALE_VM_1 --cwd /root/tmp-server2 -- python3 -m http.server --bind 127.0.0.1 8000
+```
+
+And run the tailscale funnel process in another terminal:
+
+```bash
+lxc exec $TAILSCALE_VM_1 -- tailscale funnel 8000
+```
+
+Example output:
+
+```text
+Funnel is not enabled on your tailnet.
+To enable, visit:
+
+         https://login.tailscale.com/f/funnel?node=1111B111111111T1L
+```
+
+Funnel is not enabled on a tailnet by default.
+Please configure it now by navigating to the url printed by the command,
+and clicking the "Enable" button to confirm.
+
+Once you have enabled it, the `tailscale funnel` command should print more output
+to confirm that it is now working:
+
+```text
+...
+Success.
+Available on the internet:
+
+https://tailscale-1.taild11111.ts.net/
+|-- proxy http://127.0.0.1:8000
+
+Press Ctrl+C to exit.
+```
+
+Try to navigate to the https tailnet url from your local web browser (on a machine connected to the internet, but not to the tailnet),
+and verify you see an HTML response of a directory listing including the `test-file-2.txt` created above.
+This verifies that the proxied service is indeed visible to the internet.
+
+Note: the [Tailscale funnel docs](https://tailscale.com/kb/1223/funnel#dns-propagation) state
+that you may need to wait up to 10 minutes for DNS records to propagate,
+before you can access the url.
+
+> [!WARNING]
+> In testing, the domain could not be resolved even after waiting longer than the 10 minutes.
+> The cause is to be confirmed.
+
+
+## Functionality affected by strict confinement
+
+This functionality works to some extend,
+but comes with some limitations due to strict confinement.
+
+### Tailscale login, up, and set
+
+This is about extra configuration flags to the following subcommands:
+
+- `tailscale login`
+- `tailscale up`
+- `tailscale set`
+
+All these subcommands accept many flags for configuring tailscaled,
+such as whether to accept DNS config from the admin panel,
+to use a custom control server,
+or to use a custom hostname.
+
+To see all the options supported:
+
+```bash
+lxc exec $TAILSCALE_VM_1 -- tailscale login -h
+lxc exec $TAILSCALE_VM_1 -- tailscale up -h
+lxc exec $TAILSCALE_VM_1 -- tailscale set -h
+```
+
+Due to the many possible combinations,
+testing these options is out of scope for this document.
```

Comment:
Yeah I wasn't sure about that, because login and logout works fine, but certain configuration options are either untested or unsupported. :thinking: 

Maybe I can change the title of the second section.